tests/test_importconfigs.py

```diff
@@ -40,7 +40,7 @@
 from tuna.miopen.db.tables import MIOpenDBTables, ConfigType
 from utils import CfgImportArgs
 from tuna.miopen.db.benchmark import Framework, ModelEnum, FrameworkEnum
-from tuna.miopen.db.miopen_tables import ConvolutionBenchmark
+from tuna.miopen.db.convolutionjob_tables import ConvolutionBenchmark
```

Comment:
I do like the idea of splitting these tables out into their own files

Reply:
All the tables which are used in tables.py are creating circular dependency. We need to re arrange them to resolve the same.